Example-ServiceMeshHub

```diff
@@ -0,0 +1,58 @@
+#Service Mesh Hub by Solo.io#
+
+The Service Mesh Hub is a mesh agnostic dashboard that implements the SMI specification for users to install and operate meshes. A read only version of the Service Mesh Hub can be found on https://servicemeshhub.io
+
+Service Mesh Hub does
+* Built-in installers for different meshes
+* Automatically discovers existing meshes, their configurations, running services / extensions and registers them to the Hub to manage
+* Install and manage meshes and mesh extensions  
+* Extensions tab has ecosystem tools for service mesh environments. Users can also build their own extenstions
+* Demos Tab has demo apps for trying out mesh use cases
+
+##Instructions to install Service Mesh Hub##
+Install Service Mesh Hub on your kubernetes cluster by running the following command:
+```
+$ kubectl apply -f https://raw.githubusercontent.com/solo-io/service-mesh-hub/master/install/service-mesh-hub.yaml
+```
+
+This will install the hub into the `sm-marketplace` namespace. Once the pods are running, you can open the hub by port forwarding:
+```
+$ kubectl port-forward -n sm-marketplace deploy/smm-apiserver 8080
+```
+
+Then navigate your browser to: 
+```localhost:8080```
+
+
```

Comment:
Navigate to Extensions tab and install the SMI Spec extension
http://servicemeshhub.io/extensions/default/smi